test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -260,30 +266,6 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 				gomega.Expect(createdTemplate.Template.Spec.Containers).To(gomega.BeComparableTo(updatedWl.Spec.PodSets[1].Template.Spec.Containers, ignoreContainersDefaults))
 				gomega.Expect(createdTemplate.Template.Spec.NodeSelector).To(gomega.BeComparableTo(map[string]string{"ns1": "ns1v"}))
 			})
-
-			ginkgo.By("Removing the quota reservation from the workload", func() {
```

Comment:
If now the workload is preserved, and this is an expected change we advertise in the release note, we should probably check it. I think we could use `gomega.Consistently` to prove that it continues to exist.

Reply:
Fixed